code_examples/sdk_examples/src/workshop/attester/generateAccount.ts

```diff
@@ -1,18 +1,20 @@
 import { config as envConfig } from 'dotenv'
 
-import { mnemonicGenerate, mnemonicToMiniSecret } from '@polkadot/util-crypto'
+import { mnemonicGenerate } from '@polkadot/util-crypto'
 
 import * as Kilt from '@kiltprotocol/sdk-js'
 
+// This function basically creates a keyring from a mnemonic
 export function generateAccount(mnemonic = mnemonicGenerate()): {
   account: Kilt.KiltKeyringPair
   mnemonic: string
 } {
-  const account = Kilt.Utils.Crypto.makeKeypairFromSeed(
-    mnemonicToMiniSecret(mnemonic)
-  )
-
-  return { account, mnemonic }
+  const signingKeyPairType = 'sr25519';
+  const keyring = new Kilt.Utils.Keyring({
+    ss58Format: 38,
+    type: signingKeyPairType,
+  });
+  return { account: keyring.addFromMnemonic(mnemonic) as Kilt.KiltKeyringPair, mnemonic: mnemonic };
```

Comment:
`mnemonic` can be reduced to a single value, since it is the same for both key and value.

```suggestion
  return { account: keyring.addFromMnemonic(mnemonic) as Kilt.KiltKeyringPair, mnemonic };
```